src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -340,7 +340,10 @@ class BaseOptionsSelector extends Component {
         const safeAreaPaddingBottomStyle = shouldShowFooter ? undefined : this.props.safeAreaPaddingBottomStyle;
         const textInput = (
             <TextInput
-                ref={(el) => (this.textInput = el)}
+                ref={(el) => {
+                    this.textInput = el;
+                    this.props.inputRef.current = el;
```

Comment:
inputRef can be null, so trying to access `current` could lead to an error.
```suggestion
                if (this.props.inputRef) {
                    this.props.inputRef.current = el;
                }
```

Reply:
`null` will be assigned to `current` not to `inputRef`, therefore `el` can be assigned to `.current` without a problem and need for that check